apps/unified_connector/sources/relief_web.py

```diff
@@ -279,8 +280,13 @@ def fetch(self, params):
 
         relief_url = self.URL
         total_count = 0
+        max_page_number = 0
 
         while relief_url is not None:
+            max_page_number += 1
+            if max_page_number == self.MAX_PAGE_NUMBER:
+                break
```

Comment:
```suggestion
            if max_page_number >= self.MAX_PAGE_NUMBER:
                break
            max_page_number += 1
```

Reply:
may be it should be like,
 if max_page_number > self.MAX_PAGE_NUMBER:
                break
            max_page_number += 1

to fetch data upto page 100, and break on page 100+

Reply:
Normally we start from 0 so >= works. For start: 0 , >= is okay, for start: 1 > is okay.